src/lib/sw_engine/tvgSwRenderer.cpp

```diff
@@ -114,8 +114,13 @@ struct SwShapeTask : SwTask
                   //Clip to stroke rle
                   if (shape.strokeRle && compShape->rect) rleClipRect(shape.strokeRle, &compShape->bbox);
                   else if (shape.strokeRle && compShape->rle) rleClipPath(shape.strokeRle, compShape->rle);
+             } else if (comp.method == CompositeMethod::AlphaMask) {
+                  shapeMaskingAlpha(compShape, shape.rle, compShape->rle);
+             } else if (comp.method == CompositeMethod::InvAlphaMask) {
+                  // impl
```

Comment:
please leave TODO: instead of impl
so that later we can easily grep the TODO list

Reply:
Changed